src/Meshes/geoMesh.ts

```diff
@@ -0,0 +1,790 @@
+        const distFromO = new Array<number>();
+        const distFromA = new Array<number>();
+        const distFromB = new Array<number>();
+        const vertByDist: {[key: string]: any} = {};
+        const vertData = new Array<any>();
+        let closest: number = -1;
+        let dist: number = -1;
+        for (let i = 0; i < len; i++) {
+            cartesian[i] = vertices[i].toCartesianOrigin(new IsoVector(0, 0))
+            distFromO[i] = distFrom(vertices[i], "O");
+            distFromA[i] = distFrom(vertices[i], "A");
+            distFromB[i] = distFrom(vertices[i], "B");
+            
+            if ((distFromO[i] === distFromA[i]) && (distFromA[i] === distFromB[i])) {
+                closest = 3;
+                dist = distFromO[i];
+            }
+            else if (distFromO[i] === distFromA[i]) {
+                closest = 4;
+                dist = distFromO[i];
+            }
+            else if (distFromA[i] === distFromB[i]) {
+                closest = 5;
+                dist = distFromA[i];
+            }
+            else if (distFromB[i] === distFromO[i]) {
+                closest = 6;
+                dist = distFromO[i];
+            }
+            if (distFromO[i] < distFromA[i] && distFromO[i] < distFromB[i]) {
+                closest = 2;
+                dist = distFromO[i];
+            }
+            if (distFromA[i] < distFromO[i] && distFromA[i] < distFromB[i]) {
+                closest = 1;
+                dist = distFromA[i];
+            }
+            if (distFromB[i] < distFromA[i] && distFromB[i] < distFromO[i]) {
+                closest = 0;
+                dist = distFromB[i];
+            }
+            vertData.push([closest, dist, vertices[i].x, vertices[i].y]);
+        };
+    
+        vertData.sort((a, b) => {
+            return a[2] - b[2];
+        })
+        vertData.sort((a, b) => {
+            return a[3] - b[3];
+        })
+        vertData.sort((a, b) => {
+            return a[1] - b[1];
+        })
+        vertData.sort((a, b) => {
+            return a[0] - b[0];
+        })
+        
+        for (let v = 0; v < vertData.length; v++) {
+            vertByDist[vertData[v][2] + "|" + vertData[v][3]] = [vertData[v][0], vertData[v][1], v];
+        }
+
+        this._m = m;
+        this._n = n;
+        this._vertices = vertices;
+        this._vertByDist = vertByDist;
+        this._cartesian = cartesian;
+        this._min = min;
+        this._max = max;
+    
+        return this;
+    }
+};
+
+/** Builds Polyhedron Data
+* @hidden
+*/
+
+export class PolyhedronData {
+
+    public _edgematch: any[][];
+
+    constructor (
+        public name: string,
+        public category  : string,
+        public vertex: number[][],
+        public face: number[][]
+    ) {
+
+    }
+}
+
+/**
+ * This class Extends the PolyhedronData Class to provide measures for a Geodesic Polyhedron
+ */
+export class GeoData extends PolyhedronData{

```

Comment:
GeoData can be understood as geographic instead of Geodesic. would renaming this make sense?

Reply:
Done